cmake/Options.cmake

```diff
@@ -129,22 +129,9 @@ endif()
 ####
 SET(CMAKE_CXX_FLAGS_RELEASE "-std=c++03" CACHE STRING "C++ flags." FORCE)
```

Comment:
At the same time do you mind correctly setting the C++ standard like we're doing in release/3.0 to avoid similar issues? Make sure C++11 is set when the build type is testing.

Reply:
I wasn't planning on that, as that would force the C++11 issue outside (before) the v3.0.0 branch.

Reply:
I guess I need to set it, as add compile options will not work with that flag.